### ISSUE TITLE
taxonomicName: superfamily not recognized

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#412]
![image](https://user-images.githubusercontent.com/4609956/35288291-97a30abc-0064-11e8-8467-fbe26cd764e1.png)

in animals they are always -oidea ending

**Comment by gsautter** [comment]
This one is simply about FAT not currently targeting super families, simply because they used to be rather rare before we started EJT. And even there, they look more like a "nice to have" to me than like an absolute necessity for accurate content extraction.
So because the higher taxonomy of the treatment taxa identifies correctly and unambiguously based solely on the primary higher ranks (family and order in this case).

It's surely possible to add the intermediate higher ranks as well, but I'd like to do that as pat of the planned larger FAT overhaul, which will generally render the observed ranks more flexible, as well as the patterns applied for the detection/recognition of their specific epithets. All with the main goal to better support botanical names.
